assets/src/components/map.tsx

```diff
@@ -450,18 +524,23 @@ const Map = (props: Props): ReactElement<HTMLDivElement> => {
   const defaultZoom = 13
   const [zoomLevel, setZoomLevel] = useState<number>(defaultZoom)
   const isAutoCentering: MutableRefObject<boolean> = useRef(false)
+  const [streetViewEnabled, setStreetViewEnabled] = useState<boolean>(false)
 
   const latLngs: LatLng[] = props.vehicles.map(({ latitude, longitude }) =>
     Leaflet.latLng(latitude, longitude)
   )
 
-  const autoCenteringClass = shouldAutoCenter
-    ? "m-vehicle-map-state--auto-centering"
-    : ""
+  const stateClasses = [
+    "m-vehicle-map-state",
+    shouldAutoCenter ? "m-vehicle-map-state--auto-centering" : null,
+    streetViewEnabled ? "m-vehicle-map-state--street-view-enabled" : null,
+  ]
+    .filter((c) => c !== null)
+    .join(" ")
```

Comment:
suggestion: I noticed that this code exists already in `dom.ts#className`, can we use that instead?